docs/releases/3.2.txt

```diff
@@ -655,3 +655,8 @@ Miscellaneous
   ``TransactionTestCase.assertQuerysetEqual()``, when compared to string
   values, is deprecated. If you need the previous behavior, explicitly set
   ``transform`` to ``repr``.
+
+* The ``django.core.cache.backends.memcached.MemcachedCache`` backend is
+  deprecated as it has some problems and seems to be unmaintained. Use
```

Comment:
Perhaps mention that it is the underlying client library that has the problems and is unmaintained rather than the backend in Django itself. (Otherwise people will just ask why we aren't properly maintaining Django's backend.)